models/upload.go

```diff
@@ -29,7 +30,10 @@ type Upload struct {
 	HoldTimeInMonths   int64  `gorm:"type:integer;not null;"`
 	UserName           string `gorm:"type:varchar(255);not null;"`
 	GarbageCollectDate time.Time
-	Encrypted          bool `gorm:"type:bool"`
+	Encrypted          bool   `gorm:"type:bool"`
+	FileName           string `gorm:"type:varchar(255)"`
+	FileNameLowerCase  string `gorm:"type:varchar(255)"`
+	FileNameUpperCase  string `gorm:"type:varchar(255)"`
```

Comment:
Why do we need to store lower and upper cases?

Reply:
helps to enable case-insensitive search